### PR TITLE
Update glib, libsepol, python-sphinx, python-sqlalchemy, python-virtualenv

### PR DESCRIPTION
**Pull request by henryli001**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
- Fix file section of glib-devel to include binaries that will be used by other packages
- Upgrade libsepol to enable new features that will be used by other packages 
- Add provides to python-sphinx
- Add provides to python-sqlalchemy
- Add provides to python-virtualenv

###### Change Log  

- glib: Fix file section of glib-devel
- libsepol: Upgrade libsepol
- python-sphinx: add provides
- python-sqlalchemy: add provides
- - python-virtualenv: add provides
- 
###### Does this affect the toolchain?  

YES

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/...

###### Test Methodology

- Local Build

### CODE REVIEW

File: SPECS/python-sqlalchemy/python-sqlalchemy.spec [hunk 1]
```diff
@@ -42,22 +41,32 @@ python2 setup.py install --prefix=%{_prefix} --root=%{buildroot}
 %{python2_sitelib}/*
 
 %changelog
+* Mon Feb 15 2021 Henry Li <lihl@microsoft.com> - 1.3.2-3
+- Provides python-sqlalchemy-doc
```
Comment (schmittjoseph) [inline SPECS/python-sqlalchemy/python-sqlalchemy.spec:45]: I think this changelog entry is from another spec

File: SPECS/python-sphinx/python-sphinx.spec [hunk 2]
```diff
@@ -81,6 +84,7 @@ Requires:       python3-six
 Requires:       python3-snowballstemmer
 Requires:       python3-sphinx-theme-alabaster
 Requires:       python3-sphinxcontrib-websupport
+Provides:       python3-sphinx_rtd_theme = %{version}-%{release}
```
Comment (schmittjoseph) [inline SPECS/python-sphinx/python-sphinx.spec:87]: I don't believe `python3-sphinx_rtd_theme` is provided by this package, are you sure this provide is right?

Comment (henryli001) [reply SPECS/python-sphinx/python-sphinx.spec]: You're right. Have removed the entry

File: SPECS/libsepol/libsepol.spec [hunk 3]
```diff
@@ -95,30 +96,47 @@ exit 0
 %{_libdir}/libsepol.so.1
 
 %changelog
+* Tue Feb 23 2021 Henry Li <lihl@microsoft.com> - 3.1-1
+- Upgrade libsepol to version 3.1
+
+* Fri Feb 05 2021 Joe Schmitt <joschmit@microsoft.com> - 2.9-7
+- Replace incorrect %%{_lib} usage with %%{_libdir}
```
Comment (schmittjoseph) [inline SPECS/libsepol/libsepol.spec:103]: Duplicate changelog entry?

Comment (henryli001) [reply SPECS/libsepol/libsepol.spec:103]: Fixed

**Review by schmittjoseph — APPROVED** [review]

**Pull request merged**